packages/cli/src/cmds/runCommand.ts

```diff
@@ -28,7 +28,11 @@ export default async function runCommand(
       name: `${commandPrefix}:exit-ok`,
     });
 
-    flushTelemetry(0);
+    // only run this when not in Travis builds becuase
+    // this immediately exits the test suite in Travis
+    if (!process.env.TRAVIS) {
+      flushTelemetry(0);
```

Comment:
I think flushTelemetry call (and function) should just be removed. This was needed because cli used to do `process.exit()` which killed the process before the telemetry had the chance to be flushed — this is discouraged in node precisely because it prevents any scheduled actions to run.

I believe I had changed all the places where that was done to set `process.exitStatus =` instead, which is the recommended way to set the exit status and avoids this problem. Therefore this is just not needed anymore.